crypto/cryptlib.c

```diff
@@ -120,6 +124,22 @@ void OPENSSL_cpuid_setup(void)
 # endif
 
 # if defined(_WIN32_WINNT) && _WIN32_WINNT>=0x0333
+#  ifdef OPENSSL_SYS_WIN_CORE
+TRACELOGGING_DEFINE_PROVIDER(g_hProvider, "OpenSSL",
+                              /*
+                              * Product GUID generated by MS Tool (randomly), 
+                              * Needed to send trace logs
+                              * {5ee02bf4-33cf-53cf-ddc9-f18679e677dd} 
+                              */
+                             (0x5ee02bf4, 0x33cf, 0x53cf, 0xdd, 0xc9, 0xf1, 0x86, 0x79, 0xe6, 0x77, 0xdd)
+                             );
```

Comment:
I'd say that original formatting was better. Even though this declaration looks like a function call [so that one can argue that formatting rules for function calls should apply], it's still a data declaration. And hence it's not inappropriate to format it as if it was a struct initializer. I'd even say it's more appropriate, specifically to denote that it's data declaration. Initializers are not discussed in style guide, so it's a bit of according-to-taste thing. And original was more than appropriate under the circumstances, i.e. one field is long enough to be on single line, which kind of makes field-per-line sensible choice.